hw/ip_templates/ac_range_check/ac_range_check.core.tpl

```diff
@@ -12,7 +12,7 @@ filesets:
       - lowrisc:prim:mubi
       - lowrisc:prim:all
       - lowrisc:prim:leading_one_ppc
-      - lowrisc:systems:top_racl_pkg
+      - lowrisc:virtual_constants:top_racl_pkg
```

Comment:
It is good practice to have ip_templates IPs use `- ${instance_vlnv("lowrisc:constants:top_racl_pkg")} instead of the virtual dependency, since that means it is unambiguous what the corresponding `ip_autogen` depends on.

This way only generic IPs can pick either matching pair top_racl/top packages.